EventSource.js

```diff
@@ -120,7 +120,11 @@ var EventSource = function (url, options) {
           { type: 'error', message: this.responseText });
       }
 
-      xhr.send(eventsource.OPTIONS.body || undefined);
+      if (eventsource.OPTIONS.body) {
```

Comment:
I'm not 100% sure how `xhr.send()` will handle `undefined` as an argument, but making the assumption that it does work, why not write this just as `xhr.send(eventsource.OPTIONS.body);` without the `if` statement?

Reply:
I had done that before just in case somebody sent something falsy but not undefined. But I made it an if just to be clearer.